cobra/core/model.py

```diff
@@ -316,6 +316,8 @@ def copy(self):
         except Exception:  # pragma: no cover
             new._solver = copy(self.solver)  # pragma: no cover
 
+        new._contexts = list()
```

Comment:
It's not, just realized, and got bitten by, the fact that when you copy a model, it's history manager context is also copied but that doesn't make a lot of sense to me so adding this fix here.